sonar-plugin-api/src/main/java/org/sonar/api/scan/issue/filter/FilterableIssue.java

```diff
@@ -38,8 +37,17 @@
 
   Integer line();
 
+  /**
+   * @deprecated since 5.5 use {@link #gap()}
+   */
+  @Deprecated
   Double effortToFix();
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)